.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
```

Comment:
should this be maven?

Reply:
> should this be maven?

looks like I made a mistake.
wait me for 20 minutes.

Reply:
@jamietanna done.